core_lib/web_helpers/helpers.py

```diff
@@ -0,0 +1,20 @@
+from core_lib.session.security_handler import SecurityHandler
+from core_lib.web_helpers.decorators import handle_exception
+import logging
+from core_lib.web_helpers.web_helprs_utils import WebHelpersUtils
+logger = logging.getLogger(__name__)
+
+
+def require_login_helper(request, policies, func, *args, **kwargs):

```

Comment:
This also has the else condition with exception common and will be helpful in both the frameworks rather than writing this code again
```
except Exception:
            logger.error(
                f'error while loading target page for controller entry name `{func.__name__}`', exc_info=True
            )
 return response
```

Reply:
Don't see a need for this common function